specification/dimensions/subaccountname.md

```diff
@@ -0,0 +1,41 @@
+Work item: [Issues #18](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/18)
+
+Owner: John Grubb / Michael Arkoosh
+
+Spec Content
+------------
+

```

Comment:
a few notes:

- selected block should be removed
- Sub Account Name should be Heading L1 and all others Heading L2